src/content/en/fundamentals/performance/optimizing-content-efficiency/replace-animated-gifs-with-video/index.md

```diff
@@ -125,6 +125,16 @@ to remember that your users likely won't have a reference to the GIF source like
 you will. Always adhere to your project's standards for media quality, but be
 willing to make trade-offs for performance where appropriate.
 
+If the resulting .mp4 won't open in Quicktime, Safari browser will not load it.
+You might want to run this command below:
```

Comment:
Maybe it's a good idea to specify what the comment below does by saying something like:

"You might want to run the command shown below to generate an .mp4 file that can actually run in Safari:"